cmake/Modules/FindMKLDNN.cmake

```diff
@@ -36,18 +36,22 @@ IF(NOT MKLDNN_FOUND)
     set(DNNL_MAKE_COMMAND "cmake" "--build" ".")
     include(ProcessorCount)
     ProcessorCount(proc_cnt)
-    if ((DEFINED ENV{MAX_JOBS}) AND ("$ENV{MAX_JOBS}" LESS_EQUAL ${proc_cnt}))
+    if((DEFINED ENV{MAX_JOBS}) AND ("$ENV{MAX_JOBS}" LESS_EQUAL ${proc_cnt}))
       list(APPEND DNNL_MAKE_COMMAND "-j" "$ENV{MAX_JOBS}")
       if(CMAKE_GENERATOR MATCHES "Make|Ninja")
         list(APPEND DNNL_MAKE_COMMAND "--" "-l" "$ENV{MAX_JOBS}")
       endif()
     endif()
+    if(LINUX)
+      set(ABI_NEUTRAL_FLAGS -fpreview-breaking-changes)
```

Comment:
Do we know the version of dpcpp compiler where required preview changes will land? if we know it, then I think we should have version check here to use or not to use this flag. This is in assumption that we chose not to compile with "preview" changes if we can avoid that.